port/unix_include/omrportpg.h

```diff
@@ -96,6 +96,8 @@ typedef struct OMRPortPlatformGlobals {
 	int32_t introspect_threadSuspendSignal;
 #endif /* defined(OMR_CONFIGURABLE_SUSPEND_SIGNAL) */
 #if defined(LINUX)
+	uintptr_t huge_pages_mmap_enabled;
+	uintptr_t memfd_call;
```

Comment:
Can you use the function pointer type here?
```suggestion
	memfd_call memfd_call;
```

Reply:
Yes, I would have to move the following code to `port/unix_include/omrportpg.h`
```
typedef int (*memfd_call)(const char *name, unsigned int flags);
```

That way I would have to include a cast in `port/unix/omrmmap.c`, from `PPG_memfd_call = func;` to `PPG_memfd_call = (memfd_call)func;`
And on the other hand I would be able to remove the cast in line 1146 of file `port/linux/omrvmem.c`
How does that look?

Reply:
Sounds good to me!